capstone-frontend/src/components/Home.js

```diff
@@ -1,14 +1,16 @@
 import React, { Component } from 'react';
+import { SearchUserModal } from './SearchUserModal'
 import '../App.css';
 
 class Home extends Component {
   render() {
     return (
-      <div> 
+      <div>
         <div> Your Books </div>
         <div> Your Lists </div>
         <div> Your Clubs </div>
-      </div>   
+        <SearchUserModal type='clubs'/>
```

Comment:
Why is this on the home page?

Reply:
To test the UserSearchModal for now. Will remove it in a future PR.

Reply:
I don't really know how I would automate it. The modal require user input to test it. I removed it so I doesn't go into master.